src/OpenLoco/Windows/ScenarioOptions.cpp

```diff
@@ -453,7 +454,7 @@ namespace OpenLoco::Ui::Windows::ScenarioOptions
                     break;
 
                 case Scenario::objective_type::performance_index:
-                    *(int16_t*)&*commonFormatArgs = *objectivePerformanceIndex;
+                    *(int16_t*)&*commonFormatArgs = *objectivePerformanceIndex * 10;
```

Comment:
Could you take this opportunity to use format arguments

Reply:
I considered that, but there's 16 references to `commonFormatArgs` in the file (excluding the loco_global). It seemed a little out of scope for a bugfix PR. Agreed it should be done soon, though.